src/components/transactions/eip5792/sendCalls.js

```diff
@@ -0,0 +1,254 @@
+import globalContext from '../../..';
+
+const DEFAULT_CALLS = [
+  {
+    to: '0x0c54FcCd2e384b4BB6f2E405Bf5Cbc15a017AaFb',
+    data: '0x654365436543',
+    value: '0x1234123412341234',
+  },
+  {
+    to: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
```

Comment:
Q: that's a malicious address, is it intended to have a malicious send in the batch?

Reply:
No, I was just looking for another checksummed address to test with, I'll use a different one to avoid confusion 😄